src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person: {
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+} }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {name}
+    </h2>
+    {age && (
+      <p className="Person__age">
+        I am
+        {' '}
+        {age}
+      </p>
+    )}
+    <p className="Person__partner">
+      {!isMarried && 'I am not married'}
+      {(sex === 'f' && partnerName) && `${partnerName} is my husband`}
+      {(sex === 'm' && partnerName) && `${partnerName} is my wife`}
```

Comment:
Use ternary here for conditional rendering because now it's broken
<img width="312" alt="Screenshot 2023-09-14 at 14 07 11" src="https://github.com/mate-academy/react_person/assets/80700247/e86116a1-0f70-4cf1-8cf6-e1d1d68c1284">